### PR TITLE
Fix search spinner design

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Adapt the new search spinner design.
* The new spinner location is left to the input fields cancel button, the spinner gets shown after 2 seconds.

### CODE REVIEW

**Comment by daehn** [comment]
Need to rerecord some snapshots

File: Wire-iOS/Sources/UserInterface/Components/Views/TextSearchInputView.swift [hunk 1]
```diff
@@ -86,14 +96,24 @@ public final class TextSearchInputView: UIView {
...
             searchInput.top == selfView.top
             searchInput.bottom == selfView.bottom
-            searchInput.trailing == cancelButton.leading
-            
+
+            placeholderLabel.leading == searchInput.leading
+            placeholderLabel.top == searchInput.top
+            placeholderLabel.bottom == searchInput.bottom
+            placeholderLabel.trailing == cancelButton.leading
             placeholderLabel.edges == searchInput.edges
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Components/Views/TextSearchInputView.swift]: Wouldn't those constraints create the conflict?

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Components/Views/TextSearchInputView.swift]: Oh good catch, the edges one was supposed to be removed

**Comment by daehn** [comment]
@mikeger fixed!

**Pull request merged**